test/utils/get_code_cov.sh

```diff
@@ -0,0 +1,22 @@
+if ![ lcov -v given-command > /dev/null 2>&1 ]; then
```

Comment:
For better bash resolution of "does command exist?"
```suggestion
if [ -z `which lcov` ]; then
```